app/screens/Projects/ProjectListColumnConfig.tsx

```diff
@@ -317,10 +317,12 @@ export const PROJECT_LIST_COLUMN_CONFIG: ColumnDef<IProjectItem>[] = [
 
       const handleConfirmDelete = async (event: React.MouseEvent) => {
         const projectId = row.original.projectId
+        const timestampString = new Date().toISOString()
+        const updatedName = `${row.original.projectName}_${timestampString}`
         const status = 'Archived'
```

Comment:
```suggestion
        const status = 'Deleted'
```
This could be done, and all other changes should be reverted of this file as no changes at FE level.